user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
+
+::: tip NOTE
+Strapi can either be used locally, or hosted on a server, making it available at a specific URL. No matter how you choose to use Strapi, your application will work the same way. The main difference is that, if using it locally, you will need to use your terminal to launch Strapi.
+:::
+
+![User profile](../assets/getting-started/login-page.png)
+
+To access the admin panel:
+
+1. Launch Strapi.
+2. Click on the **Open the administration** button.
+3. Enter your credentials to log in.
+4. Click on the **Log in** button. You should be redirected to the homepage of the admin panel.
+
+::: tip 💡 TIP
+Since the link which where to be redirected after clicking on **Log in** is customizable, it is possible to not be redirected to the homepage of the admin panel. To find that homepage, you can add `/admin` at the end of your Strapi application's URL.
+:::
+
+
+## Setting up your user profile
+
+If you are a new user, we recommend making sure your user profile is all set, before diving into your Strapi application. From your admin panel user profile, you are able to modify your user information, such as name, username, email or password.
+
+![User profile](../assets/getting-started/user-information-profile.png)
+
+To modify your user information:
+
+1. Click on your user name in the top right hand corner of the admin panel.
+2. Modify the information of your choice:
+
+| User information       | Instructions                                                                                             |
+| ---------------------- |----------------------------------------------------------------------------------------------------------|
+| First name             | Write your first name in the textbox.                                                                    |
+| Last name              | Write your last name in the textbox.                                                                     |
+| Email                  | Write your complete email address in the textbox.                                                        |
+| Username               | (optional) Write a username in the textbox.                                                              |
+| Password               | Write a new password in the textbox. You can click on the eye icon for the password to be shown.         |
+| Confirmation password  | Write the same new password in the textbox. You can click on the eye icon for the password to be shown.  |
+
+3. Click on the **Save** button.
+
+***
+
+Congratulations on being a new Strapi user! You're now ready to discover all the features and options that Strapi has to offer!
+
+::: tip 💡 TIP
+If you have any trouble with your Strapi experience, you can reach us through [GitHub](https://github.com/strapi/) or our [forum](https://forum.strapi.io/)! The Strapi team and the Strapi Community are always available to answer your questions or help you with anything!
```

Comment:
Here I would precise to reach GitHub for Strapi issues only and the forum for questions, how-to etc...
To avoid GitHub issues "pollution", that's the reason why we created the forum.

```diff
- The Strapi team and the Strapi Community
+ The Strapi Community and the Strapi team
```

I would put the Community first. The general idea, and what I would like it to have an active community around Strapi, and on the support.
I would like to have first the Community, and then if necessary the Strapi team is here to help. 
The scaling and healthiness of the project will be able if the Community is here to help us.

Reply:
I'll 2nd that, community first, strapi team is always second.